src/main.rs

```diff
@@ -0,0 +1,150 @@
+use crate::postgres::Postgres;
+use actyxos_sdk::event_service::{EventService, Subscription};
+use anyhow::Result;
+use backtrace::Backtrace;
+use db::{Db, DbConnection, DbEvent};
+use futures::{future::FutureExt, stream::StreamExt};
+use futures_batch::ChunksTimeoutStreamExt;
+use serde::{Deserialize, Serialize};
+use std::{env, time::Duration};
+use structopt::StructOpt;
+use tokio_compat_02::FutureExt as FutureExt02;
+use tracing::*;
+
+mod db;
+mod postgres;
+
+#[derive(StructOpt, Debug, Serialize, Deserialize)]
+#[structopt(
+    about = "Insert events into a Postgres database.\nEnvironment variables:\n  PGPASSWORD: PostgreSQL password, if not provided with --password/-p\n  AX_EVENT_SERVICE_URI: URL to connect to the Event Service. Default: http://localhost:4454/api/"
+)]
+struct Opt {
+    #[structopt(long, short)]
+    from_start: bool,
+    #[structopt(
+        long,
+        short = "w",
+        about = "If not provided, read from the PGPASSWORD environment variable"
+    )]
+    password: Option<String>,
+    #[structopt(long, short, default_value = "1024")]
+    max_batch_size: usize,
+    #[structopt(long, short)]
+    db_name: String,
+    #[structopt(long, short, default_value = "localhost")]
+    host: String,
+    #[structopt(long, short, default_value = "5432")]
+    port: u16,
+    #[structopt(long, short)]
+    username: String,
+    #[structopt(long, short, default_value = "events")]
+    table: String,
+    #[structopt(about = "Subscription set to listen to", default_value = "[{}]")]
+    subscriptions: String,
```

Comment:
isn't there a proper type in our sdk?

Reply:
I see, you should be able to directly use `Vec<Subscriptions>` here, maybe you need to provide a custom `from_str`

Reply:
Using `parse` from `structopt` worked, although I had to create an auxiliary function.